packages/itwinui-css/src/slider/slider.scss

```diff
@@ -40,7 +40,7 @@ $tick-height: var(--iui-size-s); // 12px
 
 @mixin iui-slider-rail {
   position: absolute;
-  background-color: var(--iui-color-background-border);
```

Comment:
Changed so the fill stands out stronger in high contrast modes.

**Old, using background border:**
![Screen Shot 2022-09-15 at 2 49 49 PM](https://user-images.githubusercontent.com/849817/190485547-f5869ba1-4b33-4515-8665-bf2fce4ad92c.png)

**New, using background 5:**
![Screen Shot 2022-09-15 at 2 50 19 PM](https://user-images.githubusercontent.com/849817/190485623-026914fa-e78b-4484-bde1-a020d38a9aa6.png)